modules/eks/cluster/README.md

```diff
@@ -12,43 +12,95 @@ Here's an example snippet for how to use this component.
 ```yaml
 components:
   terraform:
-    eks:
+    eks/cluster:
       vars:
         enabled: true
-        cluster_kubernetes_version: "1.21"
-        availability_zones: ["us-west-2a", "us-west-2b", "us-west-2c"]
+        name: eks
+        iam_primary_roles_tenant_name: core
+        cluster_kubernetes_version: "1.25"
+        availability_zones: ["us-east-1a", "us-east-1b", "us-east-1c"]
+        aws_ssm_agent_enabled: true
+        allow_ingress_from_vpc_accounts:
+          - tenant: core
+            stage: auto
+          - tenant: core
+            stage: corp
+          - tenant: core
+            stage: network
+        public_access_cidrs: []
+        allowed_cidr_blocks: []
+        allowed_security_groups: []
+        enabled_cluster_log_types:
+          - api
+          - audit
+          - authenticator
+          - controllerManager
+          - scheduler
         oidc_provider_enabled: true
-        public_access_cidrs: ["72.107.0.0/24"]
-        managed_node_groups_enabled: true
-        node_groups: # null means use default set in defaults.auto.tf.vars
-          main:
-            # values of `null` will be replaced with default values
-            # availability_zones = null will create 1 auto scaling group in
-            # each availability zone in region_availability_zones
-            availability_zones: null
-
-            desired_group_size: 3 # number of instances to start with, must be >= number of AZs
-            min_group_size: 3 # must be  >= number of AZs
-            max_group_size: 6
-
-            # Can only set one of ami_release_version or kubernetes_version
-            # Leave both null to use latest AMI for Cluster Kubernetes version
-            kubernetes_version: null # use cluster Kubernetes version
-            ami_release_version: null # use latest AMI for Kubernetes version
-
-            attributes: []
-            create_before_destroy: true
-            disk_size: 100
-            cluster_autoscaler_enabled: true
-            instance_types:
-              - t3.medium
```

Comment:
Shouldn't we still provide an example of how to use node groups? Since node groups are still supported. We need to be able to reference alternative configurations. perhaps a separate example?